### PR TITLE
Add textmasters parameters on translation creation

### PR DESCRIPTION
**Pull request by LaetitiaRiffaud**

### CODE REVIEW

**Comment by Pitoune** [comment]
👍

File: Translation/TranslationManager.php [hunk 1]
```diff
@@ -81,6 +82,7 @@ public function create(
             ->setOptions($options)
             ->setWorkTemplate($workTemplate)
             ->setCallback($this->generateProjectCallback())
+            ->setTextmasters($textmasters)
```
Comment (cdaguerre) [inline Translation/TranslationManager.php]: Are we sure that an empty array won't fail on TM's side?

Comment (LaetitiaRiffaud) [inline Translation/TranslationManager.php]: I can add a functional test in `textmaster-api` repository, what do you think ?

Comment (cdaguerre) [inline Translation/TranslationManager.php]: Yes please

**Pull request merged**